### ISSUE TITLE
Head isn't centered in full version of logo

### ISSUE THREAD

**Issue by ForNeVeR** [codingteam/logo#9]
While messing with the logo in inkscape, I suddenly found that the "head" is not centered inside of the whole background rectangle. Its left/right/top margins are not equal.

Is that intentional or is that something that needs to be fixed? For the time being, I've centered it inside of a square for [our main page](https://github.com/codingteam).

**Comment by rexim** [comment]
@ForNeVeR nah, I usually center things by my eyes. :D I'll try to find some time in the near future to fix that. Thanks for filing this issue.

**Comment by ForNeVeR** [comment]
Note that I've added [a branch](https://github.com/codingteam/logo/tree/feature/9-square) with a squared centered logo variant. Don't know what to do with that though \~_~

**Comment by rexim** [comment]
@ForNeVeR make a Pull Request. :)

**Comment by rexim** [comment]
@ForNeVeR I have a similar repo for @tsoding https://github.com/tsoding/brand

And I usually keep everything there.
